x/collection/keeper/msg_server_test.go

```diff
@@ -1,11 +1,12 @@
 package keeper_test
 
 import (
-	abci "github.com/tendermint/tendermint/abci/types"
+	"fmt"
 
 	sdk "github.com/Finschia/finschia-sdk/types"
 	"github.com/Finschia/finschia-sdk/x/collection"
 	"github.com/Finschia/finschia-sdk/x/token/class"
+	abci "github.com/tendermint/tendermint/abci/types"
```

Comment:
```suggestion
	abci "github.com/tendermint/tendermint/abci/types"

	sdk "github.com/Finschia/finschia-sdk/types"
	"github.com/Finschia/finschia-sdk/x/collection"
	"github.com/Finschia/finschia-sdk/x/token/class"
```

recommend to separate local packages from third-party packages